### PR TITLE
Add the ulx tpa command

### PR DESCRIPTION
**Pull request by wrefgtzweve**

### CODE REVIEW

**Comment by wrefgtzweve** [comment]
image
image
image

**Comment by wrefgtzweve** [comment]
Doesn't have a cooldown system yet

**Comment by wrefgtzweve** [comment]
image
image

Added configurable delay option and decline cooldown system.

**Comment by wrefgtzweve** [comment]
Should be done now. Hard to bugtest as I cannot use the command on bots, only on myself

**Comment by wrefgtzweve** [comment]
Might have a real smart way of doing this a lot quicker, trying that now

**Comment by wrefgtzweve** [comment]
Should be completely done now, hard to bugtest but I tested as much as I could
image

**Review by brandonsturgeon — COMMENTED** [review]
Just some styling concerns. As long as it works I'm fine with this structure

File: lua/ulx/modules/sh/cfc_tpa.lua [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    local target = targetPlayers[1]
+
+    if target:GetInfoNum( "cfc_tpa_disable", 0 ) == 1 then
+        CFCNotifications.sendSimple( "tpaNotAllowed", "TPA", "This player has tpa requests disabled.", callingPlayer )
+        return
+    end
+
+    if target.cfcTpaCooldownDecline and target.cfcTpaCooldownDecline[ callingPlayer ] and target.cfcTpaCooldownDecline[ callingPlayer ] > CurTime() then
+        CFCNotifications.sendSimple( "tpaDeclineCooldown", "TPA", "You cannot request a TPA to that player yet.", callingPlayer )
+        return
```
[the remark this answers is not in the document]
Comment (wrefgtzweve) [reply lua/ulx/modules/sh/cfc_tpa.lua:24]: Doing this will add like 40 lines extra, not worth it imo the entire file is 108 long

**Comment by wrefgtzweve** [comment]
Tested and works as far as i can see

File: lua/ulx/modules/sh/cfc_tpa.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (brandonsturgeon) [reply lua/ulx/modules/sh/cfc_tpa.lua:32]: i think ease of reading is more important than length of file, but it's not a big deal at all, you can leave it

**Review by brandonsturgeon — APPROVED** [review]

**Pull request merged**